lib/cluster.js

```diff
@@ -684,10 +684,12 @@ function workerInit() {
 
   Worker.prototype.destroy = function() {
     this.exitedAfterDisconnect = true;
-    if (!this.isConnected()) process.exit(0);
-    var exit = process.exit.bind(null, 0);
-    send({ act: 'exitedAfterDisconnect' }, () => process.disconnect());
-    process.once('disconnect', exit);
+    if (!this.isConnected()) {
+      process.exit();
```

Comment:
Ah, right... good point :-) 

@yorkie ... this should be `process.exit(0)` and below